### ISSUE TITLE
Border-color broken for ie9/10

### ISSUE THREAD

**Issue by doochik** [css/csso#134]
input
```css
.border-color {
    border-color: rgba(255,255,255,0) rgba(255,255,255,0) rgba(255,255,255,0) #fd0;
}
```

broken output
```css
.border-color{border-color:rgba(255,255,255,0)rgba(255,255,255,0)rgba(255,255,255,0)#fd0}
```

**Comment by a-koptsov** [comment]
[#232](https://github.com/css/csso/pull/232)

**Comment by lahmatiy** [comment]
Fixed by #232
